### ISSUE TITLE
Schema Compare Apply fails with an error

### ISSUE THREAD

**Issue by rickhopkins** [microsoft/azuredatastudio#24166]
Steps to Reproduce:

1. Right click on database project and click "Update project from database"
2. View changes and click aply

<img width="676" alt="image" src="https://github.com/microsoft/azuredatastudio/assets/1536701/05e0460a-d09d-4fcd-902f-9cff4b10de1f">

**Comment by dzsquared** [comment]
@rickhopkins - can you summarize or share the changes that would be applied to the project?

**Comment by rickhopkins** [comment]
<img width="1256" alt="image" src="https://github.com/microsoft/azuredatastudio/assets/1536701/31cb2473-e7c1-42ab-b30d-19438e9c27e7">
<img width="1304" alt="image" src="https://github.com/microsoft/azuredatastudio/assets/1536701/9b54e980-4046-4937-8242-6d0d3fb03696">
<img width="692" alt="image" src="https://github.com/microsoft/azuredatastudio/assets/1536701/2fd934c7-99ec-4d84-b269-5fd5b330d492">

**Comment by rickhopkins** [comment]
> @rickhopkins - can you summarize or share the changes that would be applied to the project?

I put screenshots up above. My workaround has been to delete the affected files, re-run the compare, and then apply. This seems to fix the issue, but sometimes I have quite a few more errors and it would be nice if it could apply properly. Also, it will modify the first file and really seems to mangle it... from the example above this is the Staff_Email file...

<img width="592" alt="image" src="https://github.com/microsoft/azuredatastudio/assets/1536701/b173ed62-b09b-416b-8680-dcc322fb3a05">
